src/vendor/webauthn.js

```diff
@@ -0,0 +1,185 @@
+"use strict";
+define(function () {
+
+  var webauthn = (function () {
+
+    const webauthnDB = (function () {
```

Comment:
Yes, const will not work in IE10.
Currently webauthn.js file was refactored and the line with `const` had been removed.